src/xdocs/cmdline.xml.vm

```diff
@@ -96,26 +96,27 @@ java -D&lt;property&gt;=&lt;value&gt;  \
           <code>-s</code> option. Default value is 8.
         </li>
         <li>
-          <code>-t, --tree</code> - Prints Abstract Syntax Tree(AST) of the checked file. The option
-          cannot be used other options and requires exactly one file to run on to be specified.
+          <code>-t, --tree</code> - This option is used to display the Abstract Syntax Tree (AST)
+          of the specified file. It can only be used on a single file and cannot be combined
+          with other options.
         </li>
         <li>
-          <code>-T, --treeWithComments</code> - Prints Abstract Syntax Tree(AST) with comment nodes
-          of the checked file. The option cannot be used with other options and requires exactly
-          one file to run on to be specified.
+          <code>-T, --treeWithComments</code> - This option is used to display the Abstract Syntax
+          Tree (AST) with comment nodes of the specified file. It can only be used on a single
+          file and cannot be combined with other options.
         </li>
         <li>
-          <code>-J, --treeWithJavadoc</code> - Prints Abstract Syntax Tree(AST) with Javadoc nodes
-          and comment nodes of the checked file. Attention that line number and columns will not be
-          the same as it is a file due to the fact that each javadoc comment is parsed separately
-          from java file. The option cannot be used with other options and requires exactly one
-          file to run on to be specified.
+          <code>-J, --treeWithJavadoc</code> - This option is used to display the Abstract Syntax
+          Tree (AST) with Javadoc nodes of the specified file. It can only be used on a single
+          file and cannot be combined with other options. Attention that line number and columns
```

Comment:
```suggestion
          file and cannot be combined with other options. Note that line number and columns
```

Reply:
Please change this in all cases.

Reply:
![image](https://github.com/checkstyle/checkstyle/assets/148562451/13572fae-de12-4e15-b53f-936a595bea31)

Updated for both cases